Functions/xASL_init_LongitudinalRegistration.m

```diff
@@ -141,6 +147,11 @@
         IndexFirstSubject = find(VolumeList(:,2)==min(VolumeN));
         iSubj = xASL_adm_ConvertSubjSess2Subj_Sess(x.dataset.nSessions, IndexFirstSubject);
         SubjectID_FirstVolume = x.SUBJECTS{iSubj};
+    else
+        % Default values to make sure output does not crash

```

Comment:
Not sure if big fat warnings are better than normal chill and relaxed warning 😆 but yes, let's add a warning.

Reply:
I'm also fine with adding an `error`, then we at least know why and where it crashes and it's a more predictable use-case, instead of just letting it crash with any control. Whole exception handling thing...

Reply:
f45ff39